### PR TITLE
Courses model

### PR DESCRIPTION
**Pull request by MozdolevskyiMaksym**
### I have created Courses Model.
We can use actions to:

1. Load courses(Get method), 
2. Create a new course(Post method) 
3. Edit a course(Put method)

### CODE REVIEW

File: src/models/courses/sagas.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    },
+  };
+};
+
+export function* loadCoursesWatcher() {
+  yield takeLatest(actionTypes.FETCH_COURSES, loadCoursesWorker);
+}
+
+export function* createCourseWatcher() {
+  yield takeLatest(actionTypes.CREATE_COURSE, createCourseWorker);
```
Comment (PavelKundenko) [inline src/models/courses/sagas.js:36]: Better use takeEvery here

File: src/models/courses/sagas.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+export function* editCourseWatcher() {
+  yield takeLatest(actionTypes.EDIT_COURSE, editCourseWorker);
```
Comment (PavelKundenko) [inline src/models/courses/sagas.js:40]: Better use takeEvery here

**Review by PavelKundenko — APPROVED** [review]

**Review by Icemandov — APPROVED** [review]

File: src/models/courses/reducer.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import * as actionTypes from './types';
+
+const initialState = {
+  data: null,
```
Comment (Anastasiia0999) [inline src/models/courses/reducer.js:4]: data: []

**Review by Anastasiia0999 — CHANGES_REQUESTED** [review]

**Review by Anastasiia0999 — APPROVED** [review]

**Review by archontis-cpu — APPROVED** [review]

File: src/models/index.js [hunk 4]
```diff
@@ -1,4 +1,9 @@
 export { ApiService } from './api-service/index.js';
```
Comment (sarhan-azizov) [inline src/models/index.js:1]: this is should be a part of shared folder

File: src/models/index.js [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+export {
```
Comment (sarhan-azizov) [inline src/models/index.js:2]: let's discuss this

**Review by sarhan-azizov — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Review by Winchestery — APPROVED** [review]

**Pull request merged**